backend/api/communication/src/main/java/co/airy/core/api/communication/payload/ConversationTagRequestPayload.java

```diff
@@ -5,14 +5,15 @@
 import lombok.NoArgsConstructor;
 import lombok.NonNull;
```

Comment:
unused import at the moment. I meant to ask because I don't know about this @chrismatix checkstyle can't look for those? (never really got into java linting myself)